### ISSUE TITLE
Envelope that is unarchived does not show up in list

### ISSUE THREAD

**Issue by morremeyer** [envelope-zero/frontend#792]
**Describe the bug**
When an envelope is unarchived, it does not show up in the envelope list.
It does however still shows up in the archived list, without the archive symbol.

**To Reproduce**

1. Create an envelope
2. Archive it
1. Unarchive it

**Expected behavior**
The envelope should be shown in the envelope list.

**Comment by malfynnction** [comment]
@morremeyer I can't reproduce this, can you please verify this is still happening on the latest version?

**Comment by morremeyer** [comment]
I tried to reproduce it and found the actual issue I ran into. Quick summary:

* Unarchiving an envelope works as designed
* Unarchiving an envelope in an archived category does not unarchive the category. Therefore, the envelope will not show up in the Envelope list. However, it will show up in the Dashboard.

I think unarchiving an Envelope should also unarchive its parent category, so I think we should treat this as a backend issue.
